examples/scala/build.sbt

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
```

Comment:
Do we need the ASF license here ? @tdas

Reply:
Ah just checked the delta build.sbt and it doesnt have it will remove that one